examples/generative/ipynb/wgan_gp.ipynb

```diff
@@ -486,53 +486,53 @@
    },
    "outputs": [],
    "source": [
-    "# Optimizer for both the networks\n",
-    "# learning_rate=0.0002, beta_1=0.5 are recommened\n",
+    "# Instatiate the optimizer for both the networks\n",
```

Comment:
Instantiate

Reply:
Thanks @fchollet -> `Instantiate the optimizer for both networks` (~~the~~)